catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/CqlCommands.java

```diff
@@ -146,7 +147,11 @@ protected Filter getFilter(FilterBuilder filterBuilder) throws CQLException, Par
         filterCurrentTime = System.currentTimeMillis();
 
         if (cqlFilter != null) {
-            return CQL.toFilter(cqlFilter);
+            try {
+                return CQL.toFilter(cqlFilter);
+            } catch (CQLException e) {
```

Comment:
They are pretty much the same so you can't tell the difference but I am going to look into if the backwards compatibility issues are mitigated then I can just do ECQL.